packages/@webex/widget-cc-tasks/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "@webex/cc-tasks",
+  "version": "1.0.0",
+  "main": "dist/index.js",
+  "scripts": {
+    "build": "yarn run -T tsc --declaration true --declarationDir ./dist/types",
+    "build:src": "webpack && yarn run build",
+    "build:watch": "webpack --watch",
+    "test:unit": "jest"
+  },
+  "dependencies": {
+    "@babel/core": "^7.25.2",
+    "@babel/preset-env": "^7.25.4",
+    "@babel/preset-react": "^7.24.7",
+    "@babel/preset-typescript": "^7.25.9",
+    "@testing-library/dom": "^10.4.0",
+    "@testing-library/jest-dom": "^6.6.2",
+    "@testing-library/react": "^16.0.1",
+    "@types/jest": "^29.5.14",
+    "babel-jest": "^29.7.0",
+    "babel-loader": "^9.2.1",
+    "file-loader": "^6.2.0",
+    "jest": "^29.7.0",
+    "jest-environment-jsdom": "^29.7.0",
```

Comment:
Please move jest, testing and types dependencies to `devDependencies`. Please check all the `package.json` files.